src/flask/routes/chef.py

```diff
@@ -0,0 +1,62 @@
+import json
+
+from flask import Blueprint, request
+from injector import Injector
+
+from src.api.controllers.chef_controller import ChefController
+from src.constants.audit import ExternalUsers
+from src.core.di_config import DiProviders
+from src.flask.schemas.chef_schema import ChefSchema
+from src.lib.entities.sqlalchemy_orm_mapping import Chef
+
+chef_blueprint = Blueprint("chef_routes", __name__)
```

Comment:
Could you wrap everything into a setup function like this:

```
def setup_chef_routes(ioc, ):
   chef_blueprint = ....
   chef_schema = ....

  chef_blueprint.route(...)
  def add...

 return {'chef_blueprint': chef_blueprint, ....}
```

That way we can reuse ioc instance and any other element

Reply:
done.